art/psa-runtime.ascii-art

```diff
@@ -0,0 +1,11 @@
+      MBS           IAS      Verifier
+       | read        |          |
+       | measurement |          |
```

Comment:
This should either be plural (measurements) or be in a "loop i" that, for each TE, reads the relevant measurement.

Reply:
addressed in 232ae237e